spec/features/course/experience_points/disbursement_spec.rb

```diff
@@ -61,10 +61,9 @@
         find(content_tag_selector(student_to_award_points)).find('input.points_awarded').set '100'
         find(content_tag_selector(student_to_set_zero)).find('input.points_awarded').set '00'
 
-        # ExperiencePointsRecord is not created when points_awarded is zero
         expect do
           click_button I18n.t('course.experience_points.disbursement.new.submit')
-        end.to change(Course::ExperiencePointsRecord, :count).by(1)
+        end.to change(Course::ExperiencePointsRecord, :count).by(2)
```

Comment:
Shall we also test the boundary case where experience_points_awarded is negative, and not expected the change the `:count` here?